### PR TITLE
removed startDate and updated temporal coverage

### LINKED ISSUE #3017: Remove startDate from fires and update dates in Fire descriptions
- [x] Remove startDate from fire layers
- [x] Update the dates in the Fire measurements descriptions

### PR DESCRIPTION
**Pull request by minniewong**
## Description

Fixes #3017  .

## Further comments

If this is a relatively large or complex change, start a discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

## Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/master/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

@nasa-gibs/worldview

### CODE REVIEW

**Comment by minniewong** [comment]
Awaiting FIRMS vectors from GIBS

File: config/default/common/config/wv.json/layers/modis/terra/MODIS_Terra_Thermal_Anomalies_Day.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
       "product": "MOD14",
       "type": "vector",
       "transition": "true",
-      "startDate": "2012-05-08",
+      "wrapadjacentdays": false,
```
Comment (jasontk19) [inline config/default/common/config/wv.json/layers/modis/terra/MODIS_Terra_Thermal_Anomalies_Day.json:18]: Do we need to define this property with a `false` value or can we just leave it out entirely?

Comment (minniewong) [reply config/default/common/config/wv.json/layers/modis/terra/MODIS_Terra_Thermal_Anomalies_Day.json:18]: not sure! it was in there as 'false' for some layers, so i just left it at that. We can delete it if it makes more sense

Comment (jasontk19) [reply config/default/common/config/wv.json/layers/modis/terra/MODIS_Terra_Thermal_Anomalies_Day.json:18]: Let's go ahead and remove it from all layers where it's defined as false.  I'm going through all of our configs/properties while preparing for the vis metadata work and trying to remove superfluous config properties like this.  

If the behavior is the same without it, I think it's cleaner to leave it out entirely 👍

Comment (minniewong) [reply config/default/common/config/wv.json/layers/modis/terra/MODIS_Terra_Thermal_Anomalies_Day.json]: great - removed!

**Pull request merged**